alchemist-web-renderer/build.gradle.kts

```diff
@@ -128,3 +128,13 @@ publishing.publications {
         }
     }
 }
+
+publishing {
+    publications {
+        publications.withType<MavenPublication>().configureEach {
+            if ("OSSRH" !in name) {
+                artifact(tasks.javadocJar)
+            }
+        }
+    }
+}
```

Comment:
This is okay, but maybe it's better in the main `build.gradle.kts`, applied to all "multiplatform" plugins? So that if we add another MP project its publication gets configured transparently. What do you think?

Reply:
Oh, right, that's my bad